src/main/java/com/mitrais/rnd/FlightManagementSystem/service/impl/DestinationServiceImpl.java

```diff
@@ -22,4 +23,14 @@ public void save(String name) throws EntityExistsException {
         }
         repository.save(new Destination(null, name, UserContextHolder.getUserContext().getId()));
     }
+
+    @Override
+    public List<Destination> getAvailableDestination() {
+        return repository.findAll();
+    }
+
+    @Override
+    public Destination getDestinationCityByName(String name) throws EntityNotFoundException {
+        return repository.findByName(name).orElseThrow();
```

Comment:
sedikit saran supaya nanti di view tinggal nampilin error messagenya aja
```suggestion
        return repository.findByName(name).orElseThrow(() -> new EntityNotFoundException("add the error message here"));
```